src/dtgtk/thumbtable.c

```diff
@@ -1061,22 +1061,31 @@ static gboolean _event_scroll_compressed(gpointer user_data)
     // half shown if scrollbar used)
     int move = table->thumb_size * delta;
 
-    // if the top thumb row is only partially visible, then realign first
-    const int partial_height = table->thumbs_area.y % table->thumb_size;
-    if(partial_height)
-    {
-      if(delta < 0)
-      {
-        move = partial_height;
-      }
-      else
+    // clicky scroll wheels generate integer clicks, scroll by one thumb size:
+    if (fabs(delta) == 1.0) {
```

Comment:
`fabsf(delta) == 1.0f` since delta is `float`

Reply:
Good catch, has been resolved. Thank you!